packages/api/src/command/medical/patient/consolidated-get.ts

```diff
@@ -160,7 +242,10 @@ export async function getConsolidated({
     });
     const hasResources = bundle.entry && bundle.entry.length > 0;
     const shouldCreateMedicalRecord = conversionType && conversionType != "json" && hasResources;
-    const startedAt = patient.data.consolidatedQuery?.startedAt;
+    const currentConsolidatedProgress = patient.data.consolidatedQueries?.find(
+      q => q.requestId === requestId
+    );
+    const startedAt = requestId ? currentConsolidatedProgress?.startedAt : undefined;
```

Comment:
nit: why checking `requestId`? This is already baked in `currentConsolidatedProgress`, right?